skimage/exposure/exposure.py

```diff
@@ -33,9 +75,15 @@ def histogram(image, nbins=256):
     ----------
     image : array
         Input image.
-    nbins : int
+    nbins : int, optional
         Number of bins used to calculate histogram. This value is ignored for
         integer arrays.
+    source_range : string or tuple, optional
+        'image' determines the range from the input image.
+        'dtype' determines the range from the expected range of the images
+        of that data type.
```

Comment:
You should also specify `tuple of int` and what that tuple means. =)

Reply:
Sorry, tuple of float! Should we include a tuple of float as an option for arbitrary ranges? Although I guess then we have the question of whether to ignore values outside the range or clip them to the range...

Nevertheless if you don't accept `tuple` why is it in the type signature?

Reply:
Old code, when this PR supported a custom range and I forgot to remove it. ;)